doc/OnlineDocs/modeling_extensions/gdp/solving.rst

```diff
@@ -87,6 +86,23 @@ To apply the BM reformulation within a python script, use:
 
 From the Pyomo command line, include the ``--transform pyomo.gdp.bigm`` option.
 
+Multiple Big-M (MBM) Reformulation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We also implement the multiple-parameter Big-M (MBM) approach described in literature\ [#gdp-mbm]_.
+By default, the MBM transformation will solve continuous subproblems in order to calculate M values.
+This process can be time-consuming, so the transformation also provides a method to export the M values used as a dictionary and allows for M values to be provided through the `bigM` argument.
+
+For example, to apply the transformation and store the M values, use:
+
+.. code::
+
+    mbigm = TransformationFactory('gdp.mbigm')
+    mbigm.apply_to(model)
+    M_values = mbigm.get_all_M_values()
```

Comment:
Maybe add a line here showing how these M_values can be fed to another (could be a copy of model) model?